rollup.config.mjs

```diff
@@ -7,7 +7,9 @@ export default {
   external: [
     'dgram',
     'events',
+    'module',
```

Comment:
This should not be necessary as we do not rollup the tests.

Reply:
I had put it here when i was using module in the source, that got refactored out but i forgot to remove this, thanks.